torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -487,26 +508,6 @@ struct Environment {
       }
     }
 
-    if (!retval) {
```

Comment:
Why was this moved ?

Reply:
For the same reason as values, the lookup needs to happen before we look at the builtins map.